pkg/controller/vm_webhook.go

```diff
@@ -20,6 +21,8 @@ import (
 
 // +kubebuilder:webhook:path=/mutate-v1alpha1-virtualmachine,mutating=true,failurePolicy=fail,sideEffects=None,groups=virt.virtink.smartx.com,resources=virtualmachines,verbs=create;update,versions=v1alpha1,name=mutate.virtualmachine.v1alpha1.virt.virtink.smartx.com,admissionReviewVersions={v1,v1beta1}
 
+var memoryOverhead = "256Mi"
```

Comment:
Is this confirmed or just for temp use?

Reply:
I tested VM with hugepages, and 256Mi memory overhead is enough. But with regular memory, i have no idea.